### PR TITLE
Add AC 1.10.11-1.dev

### PR DESCRIPTION
**Pull request by kaxil**
**What this PR does / why we need it**:
Create a nightly image of Airflow 1.10.11 image

1.10.11 Branch: https://github.com/astronomer/airflow/commits/v1-10-11

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] If a new distribution or Airflow version is added, please add in .circleci/generate_circleci_config.py and run the script
- [x] If a new distribution or Airflow version is added, there Dockerfile in all base image directories
- [x] If a new distribution is added, it is supported by all Airflow versions
- [x] If a new Airflow version is added, it supports all distributions
- [x] If changing an image, add applicable test in .circleci/test-airflow-image.py

### CODE REVIEW

**Comment by kaxil** [comment]
Thankfully our tests helped discover a bug :)

File: 1.10.11/alpine3.10/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	&& cp /usr/share/zoneinfo/UTC /etc/localtime \
+	&& pip3 install --no-cache-dir --upgrade snowflake-connector-python==1.9.1 \
+	&& pip3 install --no-cache-dir "${AIRFLOW_MODULE}" \
+	&& pip3 install --no-cache-dir "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.4/astronomer_airflow_scripts-0.0.4-py3-none-any.whl" \
+	&& pip3 install --no-cache-dir "astronomer-fab-security-manager~=1.2, >=1.2.2" \
+	&& apk del .build-deps py3-numpy-dev \
+	&& ln -sf /usr/bin/python3 /usr/bin/python \
+	&& ln -sf /usr/bin/pip3 /usr/bin/pip \
+	&& sed -i -e '\#https://github.com/astronomer/#d' /etc/apk/repositories \
+	&& sed -i -e 's/^run_as_user = .*/run_as_user = 100/g' /usr/lib/python3.7/site-packages/airflow/config_templates/default_airflow.cfg
```
Comment (kaxil) [inline 1.10.11/alpine3.10/Dockerfile:102]: @schnie We change the default UID for alpine image here to `100` 

The tests are passing :)

Comment (ashb) [reply 1.10.11/alpine3.10/Dockerfile:102]: I think I'd do this in a separate layer.

Also, we should probably set rbac=True default at the same time (Or have we done that in our wheels already?)

Comment (kaxil) [reply 1.10.11/alpine3.10/Dockerfile:102]: `rbac=True` is done in our Helm chart: https://github.com/astronomer/airflow-chart/blob/master/templates/configmap.yaml#L34

Comment (kaxil) [reply 1.10.11/alpine3.10/Dockerfile]: and moved the `sed` to separate layer

**Pull request closed without merging**